docker/Dockerfile

```diff
@@ -1,6 +1,6 @@
 FROM bitwalker/alpine-elixir-phoenix:1.9.0
 
-RUN apk --no-cache --update add alpine-sdk gmp-dev automake libtool inotify-tools autoconf python
+RUN apk --no-cache --update add alpine-sdk gmp-dev automake libtool inotify-tools autoconf python vim postgresql-client
```

Comment:
Maybe makes sense having two Dockerfiles, one for development and one for production running. Currently this Dockerfile is the used for the production image.

Reply:
okay, makes sense in that case to remove the extra packages added